lib.Makefile

```diff
@@ -135,7 +135,7 @@ endif
 # For building, we use the go-build image for the *host* architecture, even if the target is different
 # the one for the host should contain all the necessary cross-compilation tools
 # we do not need to use the arch since go-build:v0.15 now is multi-arch manifest
-GO_BUILD_IMAGE ?= calico/go-build
+GO_BUILD_IMAGE ?= jha48/go-build
```

Comment:
We will revert this change once https://github.com/projectcalico/go-build/pull/574 is merged.